neuralforecast/common/_modules.py

```diff
@@ -439,7 +437,6 @@ def __init__(self, d_model, embed_type="fixed", freq="h"):
         self.month_embed = Embed(month_size, d_model)
 
     def forward(self, x):
-        x = x.long()
```

Comment:
Are you sure x is always long?

Reply:
No not sure. But running the model worked fine without it, so I removed it. Worst case, let's keep it as is to avoid any potential problems